doc build error autodoc-process-signature

I have been getting this for a while and not clear why, and why just me. 


### Minimal example to reproduce issue:

```
cd Doc
make html
```

### Error message/Traceback:
```
(python3) :~/git/spacepy/Doc$ make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v3.3.1
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://numpy.org/doc/stable/objects.inv...
loading intersphinx inventory from https://docs.scipy.org/doc/scipy/reference/objects.inv...
loading intersphinx inventory from https://matplotlib.org/objects.inv...
loading intersphinx inventory from https://docs.astropy.org/en/stable/objects.inv...
[autosummary] generating autosummary for: ae9ap9.rst, autosummary/spacepy.LANLstar.LANLmax.rst, autosummary/spacepy.LANLstar.LANLstar.rst, autosummary/spacepy.ae9ap9.Ae9Data.rst, autosummary/spacepy.ae9ap9.parseHeader.rst, autosummary/spacepy.ae9ap9.readFile.rst, autosummary/spacepy.coordinates.Coords.rst, autosummary/spacepy.coordinates.quaternionConjugate.rst, autosummary/spacepy.coordinates.quaternionFromMatrix.rst, autosummary/spacepy.coordinates.quaternionMultiply.rst, ..., radbelt.rst, realtime.rst, release_notes.rst, scripts.rst, seapy.rst, spacepy.rst, spectrogram.rst, time.rst, tips.rst, toolbox.rst
/mnt/c/Users/234728/git/spacepy/spacepy/irbempy/irbempy.py:34: UserWarning: Could not import irbempylib. You appear to be building docs, so ignoring this error.
  warnings.warn('Could not import irbempylib. '
/mnt/c/Users/234728/git/spacepy/spacepy/pybats/trace2d.py:20: UserWarning: libspacepy not found. You appear to be building docs, so ignoring this error.
  warnings.warn('libspacepy not found. '
WARNING: [autosummary] failed to import 'spacepy.plot.carrington.carrington': no module named spacepy.plot.carrington.carrington
[autosummary] generating autosummary for: /mnt/c/Users/234728/git/spacepy/Doc/source/autosummary/spacepy.plot.spectrogram.rst, /mnt/c/Users/234728/git/spacepy/Doc/source/autosummary/spacepy.plot.spectrogram.rst
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 261 source files that are out of date
updating environment: [new config] 261 added, 0 changed, 0 removed
/mnt/c/Users/234728/git/spacepy/spacepy/datamodel.py:docstring of spacepy.datamodel.SpaceData.rst:5: WARNING: autosummary: failed to import SpaceData.toCDF
/mnt/c/Users/234728/git/spacepy/spacepy/datamodel.py:docstring of spacepy.datamodel.SpaceData.rst:5: WARNING: autosummary: failed to import SpaceData.toHDF5
/mnt/c/Users/234728/git/spacepy/spacepy/datamodel.py:docstring of spacepy.datamodel.SpaceData.rst:5: WARNING: autosummary: failed to import SpaceData.toJSONheadedASCII
WARNING: autodoc: failed to import method 'SpaceData.toCDF' from module 'spacepy.datamodel'; the following exception was raised:
Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 334, in safe_getattr
    return getattr(obj, name, *defargs)
AttributeError: type object 'SpaceData' has no attribute 'toCDF'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/importer.py", line 106, in import_object
    obj = attrgetter(obj, mangled_name)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 292, in get_attr
    return autodoc_attrgetter(self.env.app, obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 2242, in autodoc_attrgetter
    return safe_getattr(obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 350, in safe_getattr
    raise AttributeError(name) from exc
AttributeError: toCDF

WARNING: autodoc: failed to import method 'SpaceData.toHDF5' from module 'spacepy.datamodel'; the following exception was raised:
Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 334, in safe_getattr
    return getattr(obj, name, *defargs)
AttributeError: type object 'SpaceData' has no attribute 'toHDF5'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/importer.py", line 106, in import_object
    obj = attrgetter(obj, mangled_name)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 292, in get_attr
    return autodoc_attrgetter(self.env.app, obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 2242, in autodoc_attrgetter
    return safe_getattr(obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 350, in safe_getattr
    raise AttributeError(name) from exc
AttributeError: toHDF5

WARNING: autodoc: failed to import method 'SpaceData.toJSONheadedASCII' from module 'spacepy.datamodel'; the following exception was raised:
Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 334, in safe_getattr
    return getattr(obj, name, *defargs)
AttributeError: type object 'SpaceData' has no attribute 'toJSONheadedASCII'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/importer.py", line 106, in import_object
    obj = attrgetter(obj, mangled_name)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 292, in get_attr
    return autodoc_attrgetter(self.env.app, obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 2242, in autodoc_attrgetter
    return safe_getattr(obj, name, *defargs)
  File "/home/balarsen/miniconda3/envs/python3/lib/python3.8/site-packages/sphinx/util/inspect.py", line 350, in safe_getattr
    raise AttributeError(name) from exc
AttributeError: toJSONheadedASCII

/mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py:docstring of spacepy.plot.add_arrows:8: WARNING: Unknown interpreted text role "method".
/mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py:docstring of spacepy.plot.add_arrows:38: WARNING: Inline substitution_reference start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/carrington.py:docstring of spacepy.plot.carrington.rst:10: WARNING: autosummary: failed to import carrington
/mnt/c/Users/234728/git/spacepy/spacepy/plot/spectrogram.py:docstring of spacepy.plot.spectrogram.rst:25: WARNING: autosummary: stub file not found 'spacepy.plot.spectrogram.spectrogram'. Check your autosummary_generate setting.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/spectrogram.py:docstring of spacepy.plot.spectrogram.simpleSpectrogram:11: WARNING: Inline emphasis start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/spectrogram.py:docstring of spacepy.plot.spectrogram.simpleSpectrogram:10: WARNING: Literal block expected; none found.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/spectrogram.py:docstring of spacepy.plot.spectrogram.simpleSpectrogram:10: WARNING: Inline strong start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/spectrogram.py:docstring of spacepy.plot.spectrogram.simpleSpectrogram:10: WARNING: Inline strong start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py:docstring of spacepy.plot.utils.add_arrows:8: WARNING: Unknown interpreted text role "method".
/mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py:docstring of spacepy.plot.utils.add_arrows:38: WARNING: Inline substitution_reference start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/poppy.py:docstring of spacepy.poppy.PPro.rst:12: WARNING: autosummary: failed to import ci
/mnt/c/Users/234728/git/spacepy/spacepy/poppy.py:docstring of spacepy.poppy.PPro.rst:12: WARNING: autosummary: failed to import conf_above
/mnt/c/Users/234728/git/spacepy/spacepy/pybats/__init__.py:docstring of spacepy.pybats.add_body:41: WARNING: Inline emphasis start-string without end-string.
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.CDF.rst:203: WARNING: autosummary: failed to import CDF.backward
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import supports_int8
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_datetime_to_epoch
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_datetime_to_epoch16
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_datetime_to_tt2000
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_epoch_to_datetime
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_epoch_to_tt2000
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_epoch16_to_datetime
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_epoch16_to_tt2000
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_tt2000_to_datetime
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_tt2000_to_epoch
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import v_tt2000_to_epoch16
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import libpath
/mnt/c/Users/234728/git/spacepy/spacepy/pycdf/__init__.py:docstring of spacepy.pycdf.Library.rst:16: WARNING: autosummary: failed to import version

Extension error:
Handler <function mangle_signature at 0x7f880b753c10> for event 'autodoc-process-signature' threw an exception (exception: datetime.date.today() is not a item name in 'Create Ticktock with the current UTC date and time set to 00:00:00\n\nSimilar to date.today() with time included but in UTC and with the\ntime included (zero hours, minutes, seconds)\n\n.. versionchanged:: 0.2.2\n    This now returns the UTC day; previously it returned a Ticktock\n    UTC object, but in the local timezone, which made all conversions\n    incorrect.\n\nReturns\n=======\n    out : ticktock\n        Ticktock object with the current UTC day\n\nSee Also\n========\ndatetime.date.today()')
make: *** [Makefile:55: html] Error 2
```

A bit hard to parse but looks like `datetime.date.today()` in docs... odd if this would cause an issue. 

### OS, Python version, and dependency version information:
```
Linux-4.4.0-18362-Microsoft-x86_64-with-glibc2.10
sys.version_info(major=3, minor=8, micro=5, releaselevel='final', serial=0)
numpy=1.19.4
scipy=1.5.4
matplotlib=3.3.3
```

### Version of SpacePy
4a8455acbb5e81bf467e3b0d4704d0edb27570c5

--- a/spacepy/time.py
+++ b/spacepy/time.py
@@ -1798,7 +1798,7 @@ class Ticktock(MutableSequence):
 
         See Also
         ========
-        datetime.date.today()
+        datetime.date.today
 
         """
         warnings.warn('today() returns UTC day as of 0.2.2.',

